ffi/Cargo.toml

```diff
@@ -15,19 +15,29 @@ crate-type = ["cdylib"]
 [features]
 default = []
 tsssp = ["sspi/tsssp"]
-scard = ["sspi/scard", "dep:ffi-types"]
+scard = ["sspi/scard", "dep:ffi-types", "dep:winscard"]
 
 [dependencies]
 cfg-if = "1"
 libc = "0.2"
 num-traits = "0.2"
 whoami = "1.4"
 sspi = { path = "..", features = ["network_client", "dns_resolver"] }
-ffi-types = { path = "../crates/ffi-types", features = ["winscard"], optional = true }
+ffi-types = { path = "../crates/ffi-types", features = [
+    "winscard",
+], optional = true }
+picky-asn1-der = "0.4"
+
+winscard = { path = "../crates/winscard", features = ["std"], optional = true }
 
 # logging
 tracing = { version = "0.1" }
-tracing-subscriber = { version = "0.3", features = ["std", "fmt", "local-time", "env-filter"] }
+tracing-subscriber = { version = "0.3", features = [
+    "std",
+    "fmt",
+    "local-time",
+    "env-filter",
+] }
```

Comment:
style (non-blocking): Did you use an automated formatting tool? I think this is not a very standard way of formatting the Cargo.toml, so there is no need to change to this on purpose, but it’s not very important.